frontend/src/state/AppState.jsx

```diff
@@ -251,19 +251,32 @@ export function unabbreviateDefinitions(defs) {
   }
   return definitions;
 }
-export function urlWithSessionStorage(newCset) {
+export function getSessionStorage() {
```

Comment:
## Frontend session storage?
I think you mentioned something about this the other day. It looks like this isn't currently being utilized? Is this planned?
If we use it, how does it benefit us?
How is this session storage different than how we're already doing frontend caching?